PaddleCV/object_detection/tools/infer.py

```diff
@@ -36,6 +37,20 @@
 logging.basicConfig(level=logging.INFO, format=FORMAT)
 logger = logging.getLogger(__name__)
 
+# FIXME should this be configurable?
```

Comment:
better configurable or use save_dir of config as default.

Reply:
`save_dir` is for checkpoints, it would be confusing to save visualizations there
 also there may be hundreds of visualization files, which would clutter `save_dir`

Reply:
I'll just leave it like this for now. please add a command line flag to your PR now that the flag parser is more flexible. @heavengate

Reply:
this is now a command line flag